common/util/util.go

```diff
@@ -51,3 +52,104 @@ func GetPodsSelector(svc *v1.Service) labels.Selector {
 	}
 	return selector
 }
+
+// Contains returns true if an element is present in a iteratee.
+func Contains(in interface{}, elem interface{}) bool {
```

Comment:
We should import package go-funk instead of copying the code.
(We can't copy the code, this is a non-compliant way.😢)